definitions/3.0.0/asyncapi.json

```diff
@@ -5,8 +5,7 @@
   "type": "object",
   "required": [
     "asyncapi",
-    "info",
-    "channels"
+    "info"
```

Comment:
you need to add `operations` here 

I recommend you add some online JSON vs JSON Schema validator, 
You can bundle all these into one schema with `npm run build` and then manually check against one of the new examples.

Reply:
But `operations` is not required 🤔

Reply:
oh yeah :)